armi/reactor/converters/uniformMesh.py

```diff
@@ -296,7 +300,7 @@ def makeAssemWithUniformMesh(
             block = copy.deepcopy(sourceBlock)
             block.p.xsType = xsType
             block.setHeight(topMeshPoint - bottom)
-            block.p.axMesh = 1
+            block.p.axMesh = sourceBlock.p.axMesh
```

Comment:
I am not sure which is "right" here...do we expect this to create any problems? e.g., a new small ARMI block borrowing from a larger ARMI block with axMesh = 4.

Reply:
I think that it might be reasonable to base the axial mesh size based on the ratio of the sourceBlock height and its defined axial mesh points so that the aspect ratios remain as consistent as possible.

`--> (block.getHeight() / block.p.axMesh) = (sourceBlock.getHeight() / source.p.axMesh)`
`-->  block.p.axMesh = block.getHeight() / (sourceBlock.getHeight() / source.p.axMesh)`
`--> block.p.axMesh = (block.getHeight() / sourceBlock.getHeight()) * source.p.axMesh)`

To prevent a zero value, we can do:

`--> block.p.axMesh = (block.getHeight() / sourceBlock.getHeight()) * source.p.axMesh) or 1`

or 

`--> block.p.axMesh = math.ceil(block.getHeight() / sourceBlock.getHeight()) * source.p.axMesh))`

Reply:
I think that it might be reasonable to base the axial mesh size based on the ratio of the sourceBlock height and its defined axial mesh points so that the aspect ratios remain as consistent as possible.

`--> (block.getHeight() / block.p.axMesh) = (sourceBlock.getHeight() / source.p.axMesh)`
`-->  block.p.axMesh = block.getHeight() / (sourceBlock.getHeight() / source.p.axMesh)`
`--> block.p.axMesh = (block.getHeight() / sourceBlock.getHeight()) * source.p.axMesh)`

To prevent a zero value, we can do:

`--> block.p.axMesh = (block.getHeight() / sourceBlock.getHeight()) * source.p.axMesh) or 1`

or 

`--> block.p.axMesh = math.ceil(block.getHeight() / sourceBlock.getHeight()) * source.p.axMesh))`

Reply:
What do you think @Nebbychadnezzar / @mgjarrett ?

Reply:
That seems reasonable to me.

Reply:
Looks like @mgjarrett went back to how it is currently implemented so no change there.

Reply:
yep, sorry about the confusion. I was having a lot of mesh issues early on and I ended up making a lot of wrong turns before I figured out the root cause.